image/Makefile

```diff
@@ -0,0 +1,101 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+UBUNTU_RELEASE     = focal
+SKOPEO_VERSION     = 1.5.0
+UMOCI_VERSION      = 0.4.7
+
+IMAGE_PREFIX := podvm
+
+ARCH := $(subst x86_64,amd64,$(shell uname -m))
+
+FILES_DIR := ../image/files
+FILES      = $(shell find "$(FILES_DIR)" -type f -o -type l)
+
+ifndef IMAGE_NAME
+COMMIT := $(shell	commit=$$(git describe --match '' --dirty --always) && \
+					if [ -n "$$(git ls-files --other --exclude-per-directory .gitignore "$(FILES_DIR)")" ]; then \
+						commit="$${commit%-dirty}-dirty"; \
+					fi && \
+	                echo "$$commit")
+ifndef COMMIT
+$(error Failed to derive an image name. Explicitly define IMAGE_NAME)
+endif
+IMAGE_NAME := $(IMAGE_PREFIX)-$(COMMIT)-$(ARCH)
+endif
+IMAGE_FILE := $(IMAGE_NAME)
```

Comment:
In https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/ibmcloud/image/Makefile IMAGE_FILES is equal to IMAGE_NAME. Whereas on libvirt (https://github.com/confidential-containers/cloud-api-adaptor/blob/staging/libvirt/image/Makefile#L28) it has the suffix `.qcow2`.

Maybe:

```
IMAGE_SUFFIX ?= ""
IMAGE_FILE := $(IMAGE_NAME)$(IMAGE_SUFFIX)
```
so that the including Makefile can overwrite `IMAGE_SUFFIX`:

```
IMAGE_SUFFIX := ".qcow2"
include ../image/Makefile
```